src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -71,6 +71,7 @@ describe('Exam assessment', () => {
         });
 
         beforeEach('Create exam, exercise and submission', () => {
+            cy.login(admin);
```

Comment:
why is an admin login necessary here?
Creating exercises should done by instructors

Reply:
In this case the admin account is used to authenticate the API call to create a exercises. The exercise it not created via the UI here. But I agree that we should also use a instructor account for this call. Fixed in the latest commit